### DISCUSSION TITLE
Pull Request for Wiki to correct Dead (404) Links

### DISCUSSION THREAD

**Discussion by PJaros** [rustdesk/rustdesk#7088] [General]
I've found some dead links in the wiki. I've forked the wiki and now would like them to be included on this github wiki again.
Unfortunately there seems to be now way to create a pull-request for the wiki.

You can find the changes I made here: https://github.com/PJaros/rustdesk/wiki/FAQ/_compare/30128a665e5ff25ece2009c86df3e6ebe200bb9f...f6beca850cf9eb0a59cf22b13c53b072412a3185

Stackoverflow - [How can I make a pull request for a wiki page on GitHub?](https://stackoverflow.com/questions/10642928/how-can-i-make-a-pull-request-for-a-wiki-page-on-github)

**Comment by rustdesk** [comment]
Thanks a lot, I just updated manually.
